pages/mesosphere/dcos/services/edge-lb/1.5/related-documentation/release-notes/index.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: layout.pug
+navigationTitle:  Release notes
+title: Release notes
+menuWeight: 101
+excerpt: Provides release-specific information for Edge-LB
+enterprise: true
+---
+
+These Edge-LB Release Notes summarize release-specific changes that fix issues or update Edge-LB features for DC/OS&trade; Enterprise clusters.
+
+If you have DC/OS deployed in a production environment, see [Known issues and limitations](/mesosphere/dcos/services/edge-lb/1.5/related-documentation/known-limitations) to see if any potential operational changes for specific scenarios apply to your environment.
+
+# DC/OS for Edge-LB Service version 1.5.1 Release Notes
+
+Edge-LB Service version 1.5.1 was released on 17 December 2019.
+
+## Bug Fixes
+- Edge-LB TLS certificates now include the full bundle of intermediate certificates in the CC's cert. (DCOS-60770)
+- Edge-LB 1.5 is now using the correct container IP address. (DCOS-60517)
+
+## Other Changes
+- Upgraded SDK version to v0.57.2 to resolve a deployment issue.
+- Began collecting diagnostic bundle for all pools, by default.
+
+## Known issues and limitations
+- Auto Pool updates slow down as the number of templates increases, or as large changes to the Apache&reg; Mesos&reg; state occur, such as many apps starting or stopping.
+
+
+# DC/OS for Edge-LB Service version 1.5.0 Release Notes
+
+Edge-LB Service version 1.5.0 was released on 30 October 2019.
+
+## Breaking Changes
+- Templates now use [Sprig](http://masterminds.github.io/sprig/) template functions instead of custom functions. The [`env`](http://masterminds.github.io/sprig/os.html) functions are not exposed to templates.
+
+## Deprecation
+- The v1 API is now deprecated and D2iQ&reg; may remove it in a future release.
+
+## New features and capabilities
+- Auto Pools automatically start a pool from a template with values from Apache&reg; Mesos&reg; task labels.
```

Comment:
I just filed https://jira.d2iq.com/browse/D2IQ-63700 to clarify that this is the `self-service` feature that the 1.4 release notes mention is not supported. Unfortunately the feture was renamed between 1.4 and 1.5 ;(. If we could incoporate something here stating that the `Auto Pools` feature enables `self-service` pool administration here that would be great. One customer has already been confused by our changing of the nomenclature between versions ;(.

Reply:
Added the clarification to the first bullet point in line 41.